Domain/Domain/Sources/Interface/Repository/WhiteboardRepositoryInterface.swift

```diff
@@ -6,6 +6,18 @@
 //
 
 public protocol WhiteboardRepositoryInterface {
+    var delegate: WhiteboardRepositoryDelegate? { get set }
+
     /// 화이트보드를 주변에 알립니다.
     func startPublishing()
+
+    /// 주변 화이트보드를 탐색합니다.
+    func startSearching()
+}
+
+public protocol WhiteboardRepositoryDelegate: AnyObject {
+    /// 주변 화이트보드를 찾았을 때 실행됩니다.
+    /// - Parameters:
+    ///   - whiteboards: 탐색된 화이트보드 배열
+    func whiteboard(_ sender: WhiteboardRepositoryInterface, didFind whiteboards: [Whiteboard])
```

Comment:
단순 네이밍 컨벤션입니다만, 아래와 같은 네이밍은 어떨까요??
```suggestion
    func whiteboardRepository(_ sender: WhiteboardRepositoryInterface, didFind whiteboards: [Whiteboard])
```

Reply:
앗 !!!! 세심하지 못했군요 ~ 좋습니다 ~~~ 바아로 수정 ~ 하겠습니다 !! 감사띵똥